core/store/ledgerstore/ledger_store.go

```diff
@@ -228,7 +228,7 @@ func (this *LedgerStoreImp) initStore() error {
 	blockHeight := this.GetCurrentBlockHeight()
 
 	_, stateHeight, err := this.stateStore.GetCurrentBlock()
-	if err != nil {
+	if err != nil && err != scom.ErrNotFound {
```

Comment:
ditto

Reply:
The first time to get value will return not found